demos/appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/node/spotlight/SpotlightObserveTransitionsExampleNode.kt

```diff
@@ -156,4 +149,17 @@ class SpotlightObserveTransitionsExampleNode(
             }
         }
     }
+
+    /**
+     * Format a float to round down to a consistent 2 decimal places for display
+     */
+    private fun Float.toTwoPointPrecisionString(): String =
```

Comment:
can be simplified:
```
private fun roundFloatToTwoDecimals(float: Float): Double {
    return (float * 100.0).roundToInt() / 100.0
}
```

Reply:
The approach as written gives a consistent number of decimal places, which in some cases will yield a better UX than eg having decimals appear and disappear as numbers lerp smoothly through fractional values.